app/processing/calculation/get-calculation.js

```diff
@@ -1,20 +1,27 @@
 const schema = require('./calculation-schema')
 
 const getCalculationByPaymentRequestId = require('./get-calculation-by-payment-request-id')

```

Comment:
This function needs splitting up into:

- validateCalculation
- mapCalculation
- and then this function calling them
- need tests for other files

Reply:
Don't think it is necessary, as we are not repeating anywhere and it is direct extraction of values.